scripts/script_utils.js

```diff
@@ -274,15 +275,28 @@ function run_suite(tests) {
     }
 }
 
-run_suite([
-    [resolve`a/b/c`, `${process.cwd()}/a/b/c`],
-    [resolve`${__dirname}/../cpp/perspective`, `${process.cwd()}/cpp/perspective`],
-    [resolve`${__dirname}/../python/perspective/dist`, _path.resolve(__dirname, "..", "python", "perspective", "dist")],
-    [resolve`${__dirname}/../cpp/perspective`, _path.resolve(__dirname, "..", "cpp", "perspective")],
-    [resolve`${__dirname}/../cmake`, _path.resolve(__dirname, "..", "cmake")],
-    [resolve`${resolve`${__dirname}/../python/perspective/dist`}/cmake`, _path.resolve(_path.resolve(__dirname, "..", "python", "perspective", "dist"), "cmake")],
-    [resolve`${resolve`${__dirname}/../python/perspective/dist`}/obj`, _path.resolve(_path.resolve(__dirname, "..", "python", "perspective", "dist"), "obj")]
-]);
+if (isWin){
```

Comment:
@texodus can you take a look at this, the comments seem to suggest that this was supposed to work but it does not on windows